examples/snowflake/service/Ballerina.toml

```diff
@@ -0,0 +1,16 @@
+[[platform.java11.dependency]]
+groupId = "net.snowflake"
+artifactId = "snowflake-jdbc"
+version = "3.13.5"
+
+[[dependency]]
+org = "ballerinax"
+name = "java.jdbc"
+version = "1.0.0"
+repository = "local"
+
+[[dependency]]
+org = "ballerina"
+name = "sql"
+version = "1.0.0"
+repository = "local"
```

Comment:
These two entries don't need after we release the connectors. Let's remove them after the release